budget.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
This file is currently just a slight modification of the example budget file from sitepeed.io. The values in this file should be evaluated.